scripts/cangen/README.md

```diff
@@ -0,0 +1,41 @@
+# Cangen
+Cangen is a cli package used to generate C++/C files from a corresponding dbc file. This tool automates the can generation process and ensures can generation is consistent between all projects. 
+
+### Installation
+First install the cangen package through the terminal:
+
+```
+pip install racecar/scripts/cangen
+```
+The directory where you run this command does not matter, as long as the path specified leads to the directory containing the pyproject.toml for cangen.
+For example, if you are in the racecar directory already, then you could run 
+```
```

Comment:
There's a few markdown formatting conventions, like newlines, that can be automatically applied using the `markdownlint` vscode extension. Can you install it and run `> Format Document With` -> `markdownlint`? You can even set up your editor to "Format on Save" to automate this process. Like ruff for python, it keep our documentation consistent